src/apps/backend/modules/account/internal/store/account-db.ts

```diff
@@ -1,12 +1,15 @@
 import { Schema, Types } from 'mongoose';
 
+import { PhoneNumber } from '../../types';
+
 export interface AccountDB {
   _id: Types.ObjectId;
   active: boolean;
   firstName: string;
+  hashedPassword: string;
   lastName: string;
+  phoneNumber: PhoneNumber;
```

Comment:
I don't think this is right.

`PhoneNumber` is a class, and you will not get an instance of `PhoneNumber` class from Mongoose. You need to define `PhoneNumber` as `interface` in this file itself and refer that.